### PR TITLE
Added Terms Of use in Preferences Screen

### PR DESCRIPTION
**Pull request by ManognaMutyala**
## Description
Added Terms Of Use in Preferences Screen

## Related issues and discussion
#1065
 
 ## Screen-shots, if any
screenshot_20180304-012645

 
 ## Checklist
 

 
 - [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.
 - [x] Code is well documented
 - [x] Include unit tests for new functionality
 - [x] All user-visible strings are made translatable
 - [x] Code passes Travis builds in your branch
 - [x] If you have multiple commits please combine them into one commit by squashing them.
 - [x] Read and understood the contribution guidelines.

### CODE REVIEW

**Comment by Karljoones** [comment]
Has this been tested with the latest code? This screenshot is a little outdated as there has been significant work done to that screen. Please make sure that this goes under the "about" section.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -419,5 +419,8 @@
     <string name="donation_url">https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget/en</string>
     <string name="advanced_search_url" translatable="false">https://world.openfoodfacts.org/cgi/search.pl</string>
     <string name="advanced_search_title">Advanced Search &amp; Charts</string>
+    <string name="Terms">Terms</string>
+    <string name="TermsOfUse">Terms and Privacy</string>
+    <string name="terms_url">https://world.openfoodfacts.org/terms-of-use</string>
```
Comment (PrajwalM2212) [inline app/src/main/res/values/strings.xml]: Make the url string non-translatable .

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -419,9 +419,16 @@
     <string name="donation_url">https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget/en</string>
     <string name="advanced_search_url" translatable="false">https://world.openfoodfacts.org/cgi/search.pl</string>
     <string name="advanced_search_title">Advanced Search &amp; Charts</string>
+
+    <string name="Terms">Terms</string>
+    <string name="TermsOfUse">Terms and Privacy</string>
+    <string name="terms_url">https://world.openfoodfacts.org/terms-of-use</string>
```
Comment (PrajwalM2212) [inline app/src/main/res/values/strings.xml]: The url-string must be non translatable.

**Comment by ManognaMutyala** [comment]
I made sure it comes under about section!!Check out this commit-09a00b2 @Karljoones 
Thanks @PrajwalM2212  for the suggestion, I will make it nontranslatable.

**Comment by Karljoones** [comment]
Conflicts resolved :)

**Comment by Karljoones** [comment]
Merged, thanks @ManognaMutyala!

**Pull request merged**